### PR TITLE
Updated profile

### PR DESCRIPTION
**Pull request by owenventer**
Updated .github folder to improve the look of the company README file.

**Changes include:**
- Adding images and svg icons in the img directory
- Updating the README file with relevant links and branding.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+	&nbsp;
+    <a href="https://twitter.com/heliuslabs"><img src="https://img.shields.io/badge/twitter-follow_us-1d9bf0.svg?style=flat-square"></a>
+    &nbsp;
+    <a href="https://www.linkedin.com/company/heliusapi/"><img src="https://img.shields.io/badge/linkedin-connect_with_us-0a66c2.svg?style=flat-square"></a>
+    &nbsp;
+    <a href="mailto:info@helius.xyz"><img src="https://img.shields.io/badge/email-get_in_contact-blue?style=flat-square"></a>
+</p>
+
+<p align="center">
+	<a href="https://www.helius.dev/blog"><img height="25" src="/img/social/blog.svg" alt="Blog"></a>
```
Comment (tahsintunan) [inline profile/README.md]: I believe the correct src should be `../img/social/blog.svg`. Same goes for all the image refs.

Comment (owenventer) [reply profile/README.md]: I believe both would work as the`/img/social/blog.svg` path would just create an absolute path from the root directory which would still work with the current directory structure. Making it a relevant path would be a bit more flexible and I will update if you still think its a good idea.

Comment (tahsintunan) [reply profile/README.md]: The absolute path should work in theory, but I'm trying to see how this markdown file would look by switching to the rich diff, and it seems the images aren't showing correctly.

Comment (owenventer) [reply profile/README.md]: I've updated the file paths to relative paths but it still seems that the images don't show in the rich diff tab. They are however reflecting in my local fork.

**Review by tahsintunan — APPROVED** [review]

**Pull request merged**